parse-nessus-xml.py

```diff
@@ -28,10 +29,26 @@
 print(f'File size: {file_size}')
 print(f'Scan start date: {start_date}')
 
+DAEMONS = "bosh.dns|bosh.dns.adapter|bosh.dns.health|alertmanager|binding.cache|blackbox.exporter|bosh.exporter|cf.exporter"
+DAEMONS += "|concourse|docker|doomsday|firehose.exporter|gdn|gnatsd|grafana.server|java|kube.state.metrics|nessusd|nginx"
+DAEMONS += "|node.exporter|ntpd|oauth2.proxy|prometheus|pushgateway|ruby"
+DAEMONS += "|auctioneer|bbs|binding.cache|broker|cc.uploader|consul|discovery.registrar|dockerdt"
+DAEMONS += "|doppler|elasticsearch.exporter|etcd|file.server|flanneld|forwarder.agent|gorouter"
+DAEMONS += "|kube.apiserver|kube.controller.manager|kube.proxy|kube.scheduler|kubelet|locket"
+DAEMONS += "|log.cache|log.cache.cf.auth.proxy|log.cache.gateway|log.cache.nozzle|loggregator.agent|metrics.agent"
+DAEMONS += "|netmon|node|policy.server|policy.server.internal|prom.scraper|redis.server|rep"
+DAEMONS += "|reverse.log.proxy|rlp|reverse.log.proxy.gateway|rlp.gateway"
+DAEMONS += "|route.emitter|service.discovery.controller|silk.controller|silk.daemon"
+DAEMONS += "|ssh.proxy|statsd.injector|syslog.agent|tps.watcher|trafficcontroller|udp.forwarder|vxlan.policy.agent"
```

Comment:
Could this be a list instead, and you could do something like `DAEMONS.join('|') in the `re.search` calls below?  Just to help make this a bit more manageable when adding/removing things!

Reply:
:pixar-ooo:

Reply:
Yes. That would make maintenance a lot easier too. I can see already that I can remove a lot of k8s-related daemons.